tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -415,6 +415,53 @@ describe(__filename, () => {
     }));
   });
 
+  it('autofills slug when name is entered while creating collection', () => {
+    const name = "trishul's collection";
+    const root = render({ collection: null, creating: true });
+
+    typeInput({ root, name: 'name', text: name });
+
+    expect(root).toHaveState('slug', 'trishul-s-collection');
+  });
+
+  it('does not autofill slug when custom slug is entered while creating collection', () => {
+    const name = "trishul's collection";
+    const slug = 'trishul';
+    const root = render({ collection: null, creating: true });
+
+    typeInput({ root, name: 'slug', text: slug });
+    typeInput({ root, name: 'name', text: name });
+
+    expect(root).toHaveState('slug', slug);
+  });
+
+  it('autofills slug with trimmed collection name', () => {
+    const name = "trishul's collection";
+    const root = render({ collection: null, creating: true });
+    typeInput({ root, name: 'name', text: `  ${name}  ` });
```

Comment:
can you add a blank line before this line? (as you did for the previous test cases)